### PR TITLE
Legion Commander Buffs

### PR DESCRIPTION
**Pull request by POCKETMONEY99**
:) 
Even after been chain buffed legion commander cannot win games (I have never seen LC carry a game). This hero really doesn't translate well into 5v5 teamfights :). A more utility focused LC isn't going to work in OAA, there are better aura builders in OAA. Legion commander is only going to work as a traditional carry hero in OAA. 

Reduced Moment of Courage cooldown to 0.3 seconds at all levels. 

Set Overwhelming Odds mana cost to 100 at all levels (it used to be this). This will remove any potential mana issues the hero may have. THis spell is just a movespeed buff.

### CODE REVIEW

File: game/scripts/npc/abilities/legion_commander_moment_of_courage.txt [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "MaxLevel"                                            "6"
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "0.8" //OAA
+    "AbilityCooldown"                                     "1.5 1.2 0.9 0.6 0.3 0.3" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/legion_commander_moment_of_courage.txt]: Set this to 0.3 and leave the proc chance scaling like it was. (Its a similar change Axe got for his counter helix)

File: game/scripts/npc/abilities/legion_commander_overwhelming_odds.txt [hunk 2]
```diff
@@ -34,7 +34,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "damage"                                          "40 60 80 100 200 300"
+        "damage"                                          "40 80 120 160 320 480" //OAA
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/legion_commander_overwhelming_odds.txt]: Revert this, I will buff the talent to +120 dmg.

File: game/scripts/npc/abilities/legion_commander_moment_of_courage.txt (hunk 1 above)
Comment (POCKETMONEY99) [reply game/scripts/npc/abilities/legion_commander_moment_of_courage.txt]: We can try this but the proc chance is going to have to be lower then it currently is.

Comment (DarkoniusXNG) [reply game/scripts/npc/abilities/legion_commander_moment_of_courage.txt]: Legion's passive is the only thing that gives her carry potential (beside Duel, Duel is overbuffed already).

Comment (POCKETMONEY99) [reply game/scripts/npc/abilities/legion_commander_moment_of_courage.txt]: I cannot see this being balanced but its worth a try.

**Pull request closed without merging**